Tasks/SecretsManagerGetSecret/TaskParameters.ts

```diff
@@ -4,23 +4,23 @@
  */
 
 import { AWSConnectionParameters, buildConnectionParameters } from 'Common/awsConnectionParameters'
-import tl = require('vsts-task-lib/task')
+import { getInputOrEmpty, getInputRequired } from 'Common/vstsUtils'
 
 export interface TaskParameters {
     awsConnectionParameters: AWSConnectionParameters
     secretIdOrName: string
     variableName: string
     versionId: string
-    versionStage: string
+    versionStage: string | undefined
```

Comment:
Is the undefined necessary? Seems like getInputOrEmpty returns empty string as a fallback

Reply:
Good catch, forgot to remove the undefined